k-distribution/tests/profiling/pl-tutorial/imp/Makefile

```diff
@@ -1,5 +1,6 @@
 DEF=test
 EXT=test
+ITER=5
```

Comment:
I would not iterate calling kompile multiple times. Running it once with the verbose option already tells us a lot of things.
And the way it's set up right now, it's reusing the caches from the previous call.
```
Importing: Source(/opt/workspace/k-distribution/tests/profiling/pl-tutorial/imp/test.k)
Outer parsing [64 modules]                                   =  1.212s
Parse configurations [0/3 declarations]                      =  0.519s
  New scanner: IMP-RULE-CELLS                                =  0.459s
Parse rules [0/89 rules]                                     =  0.569s
```
If you really want to iterate though, you can either delete the cache directory (*-kompiled) or you can add
`--profile-rule-parsing <file>` as an option to kompile.

Reply:
Thanks for pointing out that issue, Radu!
However, the instructions in `kompile -hh` don't mention the need for a file and this message error seems a bit unclear to me:

```
$ kompile -hh
...
 --profile-rule-parsing
      Generate time in milliseconds to parse each rule in the semantics.
```

```
$ kompile -v --profile-rule-parsing test.k --main-module TEST

Parse command line options                                   =  0,014s
[Error] Critical: You have to provide exactly one main file in order to do
outer parsing.
```

Reply:
Also, the idea of iterating it multiple times is to ensure we're collecting a fair compilation time. That's a very common practice when executing performance benchmarks and essential when writing papers. Bencher can store the results of each execution, calculate their mean, and only output the result in the visual graph.

Reply:
The output for the help command is handled automatically by the `jcommnader` library here: https://github.com/runtimeverification/k/blob/master/kernel/src/main/java/org/kframework/kompile/KompileUsageFormatter.java
You could try to change the description to something like:
`Store in this file time taken in ms to parse each rule in the semantics.`